bee-message/tests/fpc_payload.rs

```diff
@@ -130,26 +119,26 @@ fn unpack_valid() {
 }
 
 #[test]
-fn packable_round_trip() {
+fn round_trip() {
```

Comment:
I would still keep this as `packable_round_trip`, I think the rest of the tests are named that.

Reply:
I agree, let's keep it for consistency